### PR TITLE
chore: changes for vscode extension

### PR DESCRIPTION
**Pull request by afzalsayed96**
Hi @antfu 

As discussed, I've tried to make changes to the vue app so a single codebase could work for both vscode and browser environments.

I've also opened a PR at https://github.com/afzalsayed96/vscode-icones/pull/1 which introduces loading built assets locally.

**List of changes**
1. Replace `navigator.writeText` with `copy-text-to-clippboard` package
2. Styled default scrollbar in `App.vue` so it looks better inside vscode
3. Introduce a `basePath` and `staticPath` runtime env variable for vscode environment

### CODE REVIEW

**Comment by afzalsayed96** [comment]
Also there was a small change required for getting `copy-text-to-clipboard` to work with Vite. I've opened a PR [here](https://github.com/sindresorhus/copy-text-to-clipboard/pull/38) to see if the author would be interested in merging the change upstream

**Comment by afzalsayed96** [comment]
Also would you be fine with changing the link in README to [vscode marketplace url](https://marketplace.visualstudio.com/items?itemName=afzalsayed96.icones), so users could directly download it in one click?

Those who wish to look at the code can then click on the repository link present on the marketplace page

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "@iconify/iconify": "^2.0.0-rc.1",
     "@vueuse/core": "^4.0.0-beta.15",
+    "copy-text-to-clipboard": "afzalsayed96/copy-text-to-clipboard",
```
Comment (antfu) [inline package.json:30]: any reason for this to be a git repo instead of an npm package?

Comment (afzalsayed96) [reply package.json:30]: https://github.com/antfu/icones/pull/19#issuecomment-703097341

**Comment by antfu** [comment]
Thanks

**Pull request merged**